tests/test_smoke.py

```diff
@@ -1317,6 +1317,24 @@ def test_scp_logs():
     run_one_test(test)
 
 
+@pytest.mark.kubernetes
+def test_symlink_latest_logs():
+    test = Test(
+        'symlink_latest_logs',
+        [
+            'sky launch -c test2 --cloud kubernetes -- echo hi || true',
+            'eval symlink_path="~/sky_logs/sky_latest"; [ -L "$symlink_path" ] || exit 1 ;'
+            'target_file=$(readlink -f "$symlink_path") || exit 1 ;'
```

Comment:
We may want to evaluate target_file maps to the log file path printed in the provision output (i.e., parse and compare against `/Users/romilb/sky_logs/sky-2024-07-25-18-27-29-449967/` from `To view detailed progress: tail -n100 -f /Users/romilb/sky_logs/sky-2024-07-25-18-27-29-449967/provision.log`)

Reply:
Added a grep line for this! A bit hacky because of the ansi escape code, but it should work (unless the log message changes in the future).